doc/rados/operations/pg-repair.rst

```diff
@@ -1,4 +1,65 @@
+============================
 Repairing PG inconsistencies
 ============================
+Sometimes a placement group might become "inconsistent". To return the
+placement group to an active+clean state, you must first determine which 
+of the placement groups has become inconsistent and then run the "pg
+repair" command on it. This page contains commands for diagnosing placement
+groups and the command for repairing placement groups that have become
+inconsistent.
+
+.. highlight:: console
+
+Commands for Diagnosing Placement-group Problems
+================================================
+The commands in this section provide various ways of diagnosing broken placement groups.
+
+The following command provides a high-level (low detail) overview of the health of the ceph cluster::
+
+   # ceph health detail
+
+The following command provides more detail on the status of the placement groups::
+
+   # ceph pg dump --format=json-pretty
+
+The following command lists inconsistent placement groups::
+
+   # rados list-inconsistent-pg {pgid} 
```

Comment:
@zdover23 This actually "# rados list-inconsistent-pg {**pool**} "

Reply:
@dzafman Is {pool} the proper argument of all these commands, or just the "rados list-inconsistent-pg" command?

Reply:
@zdover23 Just list-inconsistent-pg takes pool.  You then take any of the PGs listed from that command to give to the others.  That's is exactly the same as getting the PGs for a given pool that have "inconsistent" state.

Reply:
This change has been in for over a week. I hadn't recorded that it had been made when I made it.